Apps/ValidationTests/Android/app/CMakeLists.txt

```diff
@@ -65,7 +65,9 @@ target_link_libraries(BabylonNativeJNI
         Window
         ScriptLoader
         bgfx
-        XMLHttpRequest)
+        XMLHttpRequest
+        bx
+        bimg)
```

Comment:
why do you need this now but not before?

Reply:
I guess these dependencies were not private before. I've merged https://github.com/bkaradzic/bgfx.cmake for latest updates on 3rd party with bimg. It must have changed at that time.